context/app/all_labels.csv

```diff
@@ -0,0 +1,128 @@
+heart_celltype.l2,Adipocyte,AZ:0000001,fat cell,CL:0000136,skos:exactMatch
```

Comment:
This is sourced from https://github.com/hubmapconsortium/azimuth-annotate/blob/main/data/all_labels.csv - I've added this link to the `_get_all_labels` function as a comment.

Since it was last updated last year, I don't think this will need to be updated often.